sycl/test-e2e/NewOffloadDriver/split-per-source-main.cpp

```diff
@@ -0,0 +1,53 @@
+// RUN: %{build} -fsycl-device-code-split=per_source -I %S/Inputs -o %t.out %S/Inputs/split-per-source-second-file.cpp \
+// RUN: --offload-new-driver -fsycl-dead-args-optimization
+// RUN: %{run} %t.out
+//
+// XFAIL: hip_nvidia
```

Comment:
do we have a tracker for this XFAIL?

Reply:
Not sure. I see the PR marking this as XFAIL here: https://github.com/intel/llvm-test-suite/pull/525 But there is no related 'issue' for this.